web/packages/teleport/src/Account/Account.tsx

```diff
@@ -61,32 +58,14 @@ export default function AccountPage({ enterpriseComponent }: AccountPageProps) {
   const isSso = ctx.storeUser.isSso();
   const manageDevicesState = useManageDevices(ctx);
 
-  // Note: we are using the same logic here as the `AddDevice` component uses to
-  // determine whether to show various options.  This creates a duplication of
-  // logic, but this is a quick bug fix to make sure that we don't show a dialog
-  // that normally would require an OTP token, but is shown in a passwordless
-  // context and thus can't progress.
-  // TODO(bl-nero): When implementing a new device enrollment dialog, refactor
-  // this so that the options used by both components have the same source of
-  // truth.
-  const mfaOptions = createMfaOptions({
-    auth2faType: cfg.getAuth2faType(),
-    required: true,
-  });
-
-  const canAddPasskeys =
-    cfg.isPasswordlessEnabled() &&
-    mfaOptions.some(option => option.value === 'webauthn');
-
-  const canAddMFA = mfaOptions.some(
-    option => option.value === 'otp' || option.value === 'webauthn'
-  );
+  const canAddPasskeys = cfg.isPasswordlessEnabled();
+  const canAddMfa = cfg.isMfaEnabled();
 
   return (
     <Account
       isSso={isSso}
       canAddPasskeys={canAddPasskeys}
-      canAddMFA={canAddMFA}
+      canAddMFA={canAddMfa}
```

Comment:
Nit: we can change it here as well:
```suggestion
     canAddMfa={canAddMfa}
```